ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -45,12 +44,24 @@
               "attributes": {
                 "memoryLimitBytes": "2147483648"
               }
+            },
+            "machine-exec": {
+              "servers": {
+                "machine-exec": {
+                  "attributes": {
+                    "type": "terminal"
+                  },
+                  "protocol": "ws",
+                  "port": "4444"
+                }
+              },
+              "installers": []
             }
           },
           "recipe": {
-            "contentType": "",
-            "type": "dockerimage",
-            "content": "eclipse/che-theia:0.3.14-nightly"
+            "contentType": "application/x-yaml",
+            "type": "compose",
```

Comment:
FYI compose is not supported on openshift and kubernetes infras

Reply:
@garagatyi I changed two stack. First one for docker. Second one for openshift

Reply:
This change broke running Theia on kubernetes, see #11260. There is now no longer a stack that runs on kubernetes.